package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@manuscripts/transform",
   "description": "ProseMirror transformer for Manuscripts applications",
-  "version": "1.0.1",
+  "version": "1.0.1-LEAN-2001",
```

Comment:
shouldn't this be changed to a regular versioning

Reply:
so, there is no need for testing that first. before merging it.
CC: @mnatsheh